src/wormhole_transit_relay/transit_server.py

```diff
@@ -52,10 +52,7 @@ def describeToken(self):
     def connectionMade(self):
         self._started = time.time()
         self._log_requests = self.factory._log_requests
-        try:
-            self.transport.setTcpKeepAlive(True)
-        except AttributeError:
-            pass
+        self.transport.setTcpKeepAlive(True)
```

Comment:
I don't know Twisted's history enough to know if this change is related to this PR.  Curious here: is it that `transport` was not a subclass of `ITCPTransport` at some point?

Reply:
This is because `FakeTransport` (in Twisted) didn't actually-implement the full interface until a version _after_ the last one that supports python2 .. so now that we drop python2, we don't need this check anymore (because we can depend on a new-enough Twisted that fixes this issue).

Reply:
(The check doesn't hurt I suppose, but it becomes uncovered unless we run on an old Twisted)